docker/entrypoint_api_server.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env sh
 
+cp ./config/environment_variables-template.json ./config/environment_variables.json
```

Comment:
Perhaps we should check if the file already exists before copying over it? This way any changes locally won't get clobbered.